app/controllers/report_controller/reports/editor.rb

```diff
@@ -1233,20 +1246,23 @@ def set_record_vars(rpt)
     rpt.sortby = @edit[:new][:sortby1] == NOTHING_STRING ? nil : []  # Clear sortby if sortby1 not present, else set up array
 
     # Add in the chargeback static fields
-    if rpt.db == "Chargeback" # For chargeback, add in static fields
-      rpt.cols = ["start_date", "display_range", "vm_name"]
+    if rpt.db.starts_with? "Chargeback" # For chargeback, add in static fields
+      rpt.cols = ["start_date", "display_range"]
```

Comment:
To solve :large_orange_diamond: - Line 1250, Col 18 - Style/WordArray - Use %w or %W for an array of words. in 1250 use
rpt.cols = %w("start_date" "display_range")